src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/FrequentExpenseBuilder.java

```diff
@@ -0,0 +1,84 @@
+package ay2021s1_cs2103_w16_3.finesse.testutil;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ay2021s1_cs2103_w16_3.finesse.model.category.Category;
+import ay2021s1_cs2103_w16_3.finesse.model.frequent.FrequentExpense;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Amount;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Date;
+import ay2021s1_cs2103_w16_3.finesse.model.transaction.Title;
+import ay2021s1_cs2103_w16_3.finesse.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building FrequentExpense objects.
+ */
+public class FrequentExpenseBuilder {
```

Comment:
This class seems debatable. Note that `TransactionBuilder` was used to cover both `Expense` and `Income`. Should the same be done here? i.e. `FrequentTransactionBuilder` to cover `FrequentExpense` and the upcoming `FrequentIncome`.